Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FINUFFT"
 uuid = "d8beea63-0952-562e-9c6a-8e8ef7364055"
 author = "Ludvig af Klinteberg <ludvigak@kth.se>"
-version = "1.1.0"
+version = "2.0.0"
 
 [deps]
 finufft_jll = "c41cd5a2-72a3-5203-9076-a500b088fc82"
```

Comment:
I just realized that we also need to point out the required version of `finufft_jll` under `[compat]`.
(Tested the branch on my computer, and it failed because I had v1.1.2 intalled)
I'm not sure how strict we need to be, but it should probably be enough with `finufft_jll = "2"`